src/ducks/balance/components/AccountRow.jsx

```diff
@@ -131,11 +131,11 @@ class AccountRow extends React.PureComponent {
             totalClassName={styles.AccountRow__figure}
             currencyClassName={styles.AccountRow__figure}
           />
+          {/* color: Do not deactivate interactions with the button,
+            only color it to look disabled */}
```

Comment:
Where was syntax color broken?

Reply:
In sublime text.

Reply:
Oh ok, I didn't understand since it looked OK in Github :sweat_smile: